Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.10",
+	"GoVersion": "go1.11",
```

Comment:
I think it's ok to bump this version now, but maybe @ixdy can confirm.

Reply:
I think this field is automatically filled in by `godep` based on what version of the runtime it's using. it's pretty meaningless.